### ISSUE TITLE
js_out with import_style=closure is incompatible with grpc-web_out import_style=Typescript

### ISSUE THREAD

**Issue by bpicolo** [grpc/grpc-web#411]
Not sure what the best resolution is, but should probably raise an error if attempting to use these two together.

**Comment by pumano** [comment]
Yes, defferent import styles. I add that "note" to documentation because I found that problem too. Just use commonjs as js_out and typescript for grpc_out. 

I use commonjs for js_out and commonjs+dts for grpc-web_out. It works good.
